File: demo/src/main/java/com/bytedance/scenedemo/MainActivity.java
package com.bytedance.scenedemo;

import com.bytedance.scene.Scene;
import com.bytedance.scene.ui.SceneActivity;

import org.jetbrains.annotations.NotNull;

public class MainActivity extends SceneActivity {

    @NotNull
    @Override
    protected Class<? extends Scene> getHomeSceneClass() {
        return MainScene.class;
    }

    @Override
    protected boolean supportRestore() {
        return true;
    }

}

File: demo/src/main/java/com/bytedance/scenedemo/MainActivity.kt
package com.bytedance.scenedemo

import com.bytedance.scene.Scene
import com.bytedance.scene.ui.SceneActivity

class MainActivity : SceneActivity() {
    override fun getHomeSceneClass(): Class<out Scene> {
        return MainScene::class.java
    }

    override fun supportRestore(): Boolean {
        return true
    }
}
